src/index.css

```diff
@@ -0,0 +1,7 @@
+body {
```

Comment:
usually it's nice to have a reset css here, it saves a lot of debugging and cross browser implementation issues

Reply:
Meyer Reset added in the last commit